llvm/tools/sycl-post-link/SpecConstants.cpp

```diff
@@ -622,12 +701,20 @@ PreservedAnalyses SpecConstantsPass::run(Module &M,
   for (const auto &P : SCMetadata)
     MD->addOperand(P.second);
 
+  // Emit default values metadata only in native (default) spec constants mode.
```

Comment:
Could you please remind where the value of a JITted spec constant is taken from when it is not explicitly set by the SYCL app? (in which case it should be set to the default)

Reply:
AFAIK, it should be set explicitly. Metadata is not needed in JIT mode because the default value is stored via SPIR-V intrinsic.

Reply:
ok, thanks